openquakeplatform/openquakeplatform/irv/static/irv/js/irv_viewer.js

```diff
@@ -29,6 +29,31 @@ var widgetsAndButtons = {
                    'buttonText': 'Show Composite Indicator Chart'}
 };
 
+var operators = ['Simple sum (ignore weights)',
+                 'Weighted sum',
+                 'Average (ignore weights)',
+                 'Simple multiplication (ignore weights)',
+                 'Weighted multiplication',
+                 'Geometric mean (ignore weights)'];
+
```

Comment:
Use directly operators value inside the code could be error prone, please use an indirect approach like Array and indexes or object with a list of attributes could be a better and more robust solution.